lib/src/base-styles/_nx-tabs.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ /*
+  #Tabs
+*/
+.nx-tab-list {
+  border-bottom: $nx-border-row-dark;
+  padding-left: 0;
+}
+
+.nx-tab {
+  display: inline-block;
+  list-style: none;
+  margin-bottom: -1px;
```

Comment:
What's the purpose of this? It seem to just cover up the bottom pixed of the blue border, which could be just as easily achieved by decreasing the width of that border I would think

Reply:
This just ensures that the active NxTab border overlaps the border for the NxTabList. We can't decrease the width of the NxTabList without removing the border from all other tabs entirely. Happy for other suggestions here from @dturner32.

Reply:
> What's the purpose of this? It seem to just cover up the bottom pixed of the blue border, which could be just as easily achieved by decreasing the width of that border I would think

It seems to be gone now